tiledb/sm/query/readers/sparse_global_order_reader.cc

```diff
@@ -855,15 +919,37 @@ SparseGlobalOrderReader<BitmapType>::merge_result_cell_slabs(
     uint64_t num_cells, CompType cmp) {
   auto timer_se = stats_->start_timer("merge_result_cell_slabs");
   std::vector<ResultCellSlab> result_cell_slabs;
+  auto cmp_no_timestamps = cmp;
 
   // TODO Parallelize.
 
   // For easy reference.
-  auto dups = array_schema_.allows_dups() || consolidation_with_timestamps_;
+  const bool dups =
```

Comment:
This variable is only used once I think, otherwise still many calls to `array_schema.allows_dups()` below.

Reply:
(I don't have a strong preference either way on it, just consistency)